src/components/templates/Home/index.js

```diff
@@ -15,8 +15,14 @@ import turtle from 'assets/Images/turtle.png';
 import starFish from 'assets/Images/starfish.svg';
 import FaAngleRight from 'react-icons/lib/fa/angle-right';
 
+global.starpuff_count = 0;
```

Comment:
i'm not sure if we should introduce `global` variable, we can always extract it from firebase right?

Reply:
also use `camelCase` instead using `underscore_variable`

Reply:
@andiwinata I think this was @chengcai616's commit that @Suyinkee was trying to get reviewed?